core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/Cluster.java

```diff
@@ -649,10 +650,6 @@ public ExecutorType getExecutorType() {
         return executorType;
```

Comment:
Nit: Is the getter still needed?

Reply:
I don't think so, but should be thrown out when we get rid off the field